lensless/recon/recon.py

```diff
@@ -404,6 +404,28 @@ def get_image_estimate(self):
         """Get current image estimate as [Batch, Depth, Height, Width, Channels]."""
         return self._form_image()
 
+    def _set_psf(self, psf):
```

Comment:
can remove the "_" in beginning of file function? Unless you don't want users to use this? (convention is to begin with "_" if it's a private method)

Reply:
It is intended to be a private function. For Unrolled and pre/post changing the psf while break the reconstruction.